### DISCUSSION TITLE
running KAPE and copying image to a remote host

### DISCUSSION THREAD

**Discussion by tomee7** [CrowdStrike/psfalcon#404] [Q&A]
hello,

i am leveraging psfalcon to run and execute KAPE via Crowdstrike RTR on a remote host, Once KAPE runs and generates an image file, it zips it up on the remote host in the temp folder location.
What im trying to do is grab that file off the remote host and copy it to another host which stores the KAPE image.
would i be best to use Invoke-FalconAdminCommand to use 'put' file?

example of what i tried to do but not sure if copy path will work in this instance?

```
`
variables
$RemoteHost = "KAPE image storage host name"
$RemotePath = "C:\KAPE_Image_storage_Folder"
$LocalFilePath = "C:\Temp\KAPE_Image.zip"
 
Try {
 
    Write-Host "Copying everything to the KAPE storage host"
    Write-Log -level INFO -LogOutput "Copying everything to KAPE storage host" -Path $LogFile
 
    $upload = Invoke-FalconRtr -Command runscript -Arguments "-Raw=`"cp $LocalFilePath $RemotePath -Timeout=3600`"" -HostId $RemoteHost
    Start-Sleep -seconds 20


 #then do a check loop

````

**Comment by bk-cs** [comment]
If a file exists on one host and you're trying to copy it to another, you either need to mount a network share inside RTR (`map`) and copy it, or `get` the file, use [Receive-FalconGetFile](https://github.com/crowdstrike/psfalcon/wiki) to download it, [Send-FalconPutFile](https://github.com/crowdstrike/psfalcon/Send-FalconPutFile) to upload it as a `put` file, then `put` the file on the remote host using RTR.
